history-server.md

```diff
@@ -0,0 +1,76 @@
+# Spark History Server
+
+### Prerequisites
+
+Required software:
+* K8s cluster
+* [KUDO CLI Plugin](https://kudo.dev/docs/#install-kudo-cli) 0.7.3 or higher
+
+### Installing Spark Operator with History Server Enabled
+
+```
+kubectl kudo install ./kudo-operator \
+    -n <NAMESPACE> \
```

Comment:
did KUDO add `-n` support? I think users still need to provide `--namespace` instead

Reply:
No KUDO is not supporting `-n`, replaced it with `--namespace`. Thanks for pointing it out.